tenant/config.go

```diff
@@ -139,11 +139,16 @@ func (c *Config) calculateFQMNs() {
 			mod.Namespace = fqmn.NamespaceDefault
 		}
 
-		c.Modules[i].FQMN = c.FQMNForFunc(mod.Namespace, mod.Name, mod.Ref)
+		var err error
+		c.Modules[i].FQMN, err = c.FQMNForFunc(mod.Namespace, mod.Name, mod.Ref)
+
+		if err != nil {
+			// ???????
```

Comment:
@cohix Let me know what should happen in this case.

Reply:
I'm not @cohix  but I'd just ignore this module and log a warning that a module with an invalid fqmn was discarded.

Reply:
Hmm well discarding wouldn't be the simplest thing since we'd need to edit the array of Modules. I think we can ignore it, if it doesn't have an FQMN, then nothing will be able to run it, so it'll be unusable but not a big problem for the system as a whole.